deb/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        dh-make \
+        devscripts \
+        equivs \
+        lintian
+
+ENV DEBFULLNAME "DIGITS Development Team"
+ENV DEBEMAIL "digits@nvidia.com"
+ENV DEB_BUILD_OPTIONS nocheck
+
+ARG UPSTREAM_VERSION
+ARG DEBIAN_VERSION
+
+WORKDIR /build
+COPY tarball/* .
+RUN tar -xf *.orig.tar.gz
+WORKDIR /build/digits
+RUN dh_make -y -s -c bsd -d -t `pwd`/deb/templates \
+    -f ../*.orig.tar.gz -p digits_${UPSTREAM_VERSION}
+RUN dch --create --package digits -v $DEBIAN_VERSION "v${DEBIAN_VERSION}"
+RUN dch -r ""
+RUN cp -R deb/extras/* debian/
+RUN apt-get update && echo y | mk-build-deps -i -r debian/control
+RUN debuild --no-lintian -i -uc -us -b
+RUN debuild --no-lintian -i -uc -us -S -sa
+RUN lintian ../*.changes
+RUN mkdir -p /dist
+RUN cp ../* /dist/ || true
```

Comment:
I don't think this really matters if I'm not actually distributing the docker image. But ok.